apps/mobile/src/main.tsx

```diff
@@ -1,4 +1,4 @@
 import { AppRegistry } from 'react-native';
 import AppRoot from './app/App';
 
-AppRegistry.registerComponent('inHEARTive', () => AppRoot);
+AppRegistry.registerComponent('Mobile', () => AppRoot);
```

Comment:
This change potentially breaks Android build that is already set up and tested in Github Actions

Reply:
But previous change broke ios development :D

Reply:
Ok we need to find some solution